anti-evil-maid/README

```diff
@@ -51,13 +51,19 @@ PCR-04: 93 33 4E 81 A6 9C 80 54 D6 87 C7 FD 76 7C 6F 4C 70 FC C6 73
 the TPM SRK key used for sealing process:
 
 # systemctl restart tcsd
-# tpm_takeownership -y -z
+# tpm_takeownership -z
 
 You may want to use non-default password for the SRK key (see the discussion in
 the article referenced above), certainly if you want to save the sealed secrets
 to your internal boot partition. In that case you SHOULD NOT pass the '-z'
 argument to tpm_takeownership.
 
+Please note that non-default owner password is required (ie. do not pass '-y'
+argument to tpm_takeownership)! Without an owner password set, the single-use
+anti-replay "freshness" tokens would be useless, as an attacker could simply
+rewrite the TPM NVRAM area with previously-valid-and-captured freshness token
+hash(es).
+
```

Comment:
Now that `anti-evil-maid-install` handles taking ownership, let's get rid of this whole section 3 (except for the last paragraph).